Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor

```diff
@@ -1,6 +1,96 @@
 ﻿@page "/tabs";
+@using OpenCodeDev.Blazor.Foundation.Doc.Core.Components.Containers.Section;
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Tabs</h2>
+        <p>
+            Tabs are components that help you organize and navigate multiple documents in a single container. They can be used for switching between items in the container.
+        </p>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>
+            There are two pieces to a tabbed interface: the tabs themselves, and the content for each tab. The tabs are an component with the <Code Content="PFRhYkhlYWRlcj4="></Code>, and each item has the <Code Content="PFRhYkhlYWRlclRpdGxlPg=="></Code>. Each tab contains a link to a tab.
```

Comment:
are an component ->  are a component
See [when to use a/an](https://www.grammar-monster.com/lessons/an_or_a.htm#:~:text=Using%20%22An%22%20and%20%22A%22,-There%20is%20sometimes&text=(The%20words%20%22an%22%20and,a%20house%20in%20an%20hour.)

and each item has the ??? the what the component?
Each tab contains a link to a tab. (Remove)

Reply:
Yes 
```suggestion
            There are two pieces to a tabbed interface: the tabs themselves, and the content for each tab. The tabs are a component with the <Code Content="PFRhYkhlYWRlcj4="></Code>, and each item has the <Code Content="PFRhYkhlYWRlclRpdGxlPg=="></Code>. 
```